app/assets/stylesheets/modules/_details.scss

```diff
@@ -0,0 +1,125 @@
+.details-hero {
+  background-color: $white;
+  text-align: center;
+
+  &-title {
+    margin-bottom: em(5);
+  }
+
+  &-compliment {
+    display: inline-block;
+
+    &-refresh {
+      background: image-url("refresh.png");
+      background-repeat: no-repeat;
+      background-size: 14px;
+      display: inline-block;
+      height: em(15);
+      width: em(15);
+      margin-left: em(10);
+
+      &:hover {
+        opacity: 0.5;
+        cursor: pointer;
+      }
+    }
+  }
+
+  &-link {
+    font-size: $base-font-size/1.3;
+    font-weight: 300;
+    font-style: italic;
+    margin-bottom: $secondary-spacing;
+  }
+
+  &-text {
+    @include span-columns (6 of 12);
+    @include shift(3);
+
+    p {
+      margin-bottom: em(30);
+    }
+  }
+
+  &-yes {
+    color: $base-accent-color;
+    font-weight: 600;
+    letter-spacing: 1.3px;
+
+    &-arrow {
+      width: 0; 
```

Comment:
Properties should be ordered border-left, border-right, border-top, font-size, height, line-height, width